openedx/adg/admin_dashboard/performance/helpers.py

```diff
@@ -0,0 +1,70 @@
+"""
+Helper methods for `performance` app
+"""
+from lms.djangoapps.grades.models import PersistentCourseGrade
+from openedx.adg.admin_dashboard.performance.constants import COURSE_GRADES_REPORT_HEADER
+from openedx.core.lib.grade_utils import round_away_from_zero
+
+
+def convert_float_point_to_percentage(float_value):
```

Comment:
Can we move this function out of admin dashboard to some global utilities where it can be used across application?

Reply:
Right, Earlier it was related to grades but now yes we can move it to a common place. Thanks for pointing it out.